File: The Core/rounders.cs
int rounders(int n) {
    string num = n.ToString();
    var chars = num.ToCharArray();
    bool carry = false;
    string result = "";
    
    for(int i = chars.Length - 1; i >= 0; i--){
        var c = int.Parse(chars[i].ToString());
        
        if(carry){
            c += 1;
        }
        
        carry = false;
        
        if(c >= 5){
            carry = true;
        }
        
        if(i != 0){
            result += "0";
        } else {
            if(c == 10){
                result += "01";
            }         
            else{
                result += c.ToString();
            }
        }
    }
    
    var s = result.ToCharArray();
    Array.Reverse(s);
    result = new string(s);
    return int.Parse(result);
}

File: The Core/Loop Tunnel/32. Rounders/rounders.py
def rounders(n):
    num = str(n)
    chars = list(num)
    carry = False
    result = ""
    
    for x in range(len(chars) - 1, -1, -1):
        c = int(str(chars[x]))
        
        if(carry):
            c += 1
        
        carry = False
    
        if(c >= 5):
            carry = True
        
        if(x != 0):
            result += "0"
        else:
            if(c == 10):
                result += "01"
            else:
                result += str(c);
    
    return int(result[::-1])
